src/space.js

```diff
@@ -167,6 +167,7 @@ class Space {
    * @param     {Boolean}   opts.confidential       create a confidential thread with true or join existing confidential thread with an encKeyId string
    * @param     {Boolean}   opts.noAutoSub          Disable auto subscription to the thread when posting to it (default false)
    * @param     {Boolean}   opts.ghost              Enable ephemeral messaging via Ghost Thread
+   * @param     {String}    opts.ghostPinbot        Enable ephemeral messaging via Ghost Thread
```

Comment:
Should say something like: `Multiaddress of a ghostpinbot instance`

Reply:
oh, copied from above, thanks! fixed 👍